src/types.ts

```diff
@@ -276,6 +279,7 @@ export interface Meta {
   accumulated_weight: number;
   score: number;
   height: number;
+  validation?: string;
```

Comment:
Why do you need this?

Reply:
We don't, but it is part of the `meta` object that comes from the full-node:

https://github.com/HathorNetwork/hathor-wallet-service-sync_daemon/pull/15/files#diff-1a258b987746216dc81c746069ca5944c0f1ec848aa095a8a9f109a976dc2343R366